pom.xml

```diff
@@ -29,10 +29,9 @@
     <properties>
         <revision>3.10.0</revision>
         <changelist>-SNAPSHOT</changelist>
-        <gitHubRepo>jenkinsci/config-file-provider-plugin</gitHubRepo>
+        <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
```

Comment:
While I was here, improving this bit of logic to be consistent with the canonical format in `jenkinsci/archetypes`.